src/core/SliceRepresentation.js

```diff
@@ -186,6 +200,11 @@ export default class SliceRepresentation extends Component {
       this.validData = true;
       this.actor.setVisibility(this.currentVisibility);
 
+      // reset camera after input data is lazy-loaded
+      if (this.view && this.view.props.autoResetCamera) {
+        this.view.resetCamera();
```

Comment:
Just a note: the rewrite should cover this case without having to add extra checks for it. This can stay to make sure it works.

Reply:
> SliceRepresentation exposes I/J/K/X/Y/Z slicing properties. I wonder if we should split out the functionality into something like a SliceStackRepresentation. What do you think @jadh4v @sankhesh?
> A note for the rewrite: depending on how the types work out, I may split the ImageArrayMapper and ImageMapper into two separate components, but I haven't given it much thought just yet.

Currently we don't have a Mapper component. If you create `ImageMapper` and `ImageArrayMapper` components perhaps the `SliceRepresentation` can be oblivious to the mapper type as the slicing properties can be part of the mapper components?? In that case, we wouldn't need a `SlicerStackRepresentation`.